build.sbt

```diff
@@ -22,7 +22,15 @@ excludeLintKeys in Global ++= Set(ideSkipProject, reStartArgs)
 
 val commonSettings = commonSmlBuildSettings ++ ossPublishSettings ++ Seq(
   organization := "com.softwaremill.sttp.tapir",
-  mimaPreviousArtifacts := Set.empty, //Set("com.softwaremill.sttp.tapir" %% name.value % "0.12.21")
+  mimaPreviousArtifacts := {
+    val minorUnchanged = previousStableVersion.value.flatMap(CrossVersion.partialVersion) == CrossVersion.partialVersion(version.value)
+    val isRcOrMilestone = version.value.contains("M") || version.value.contains("RC")
```

Comment:
`sbt-dynver` treats milestones and RC as stable releases, and I guess we'd like to accept MiMa errors between `0.18.0-M1` and `0.18.0-M2`, thus the check

Reply:
What will happen then with a `0.18.0` build? Will this compare against `0.18.0-M[last]`, the previous version tag, or against `0.17.[last]`?

Reply:
Not sure, I'll check and revise it

Reply:
Checked that. It will compare against milestone:
```scala
sbt:tapir> core / previousStableVersion
[info] Some(0.18.0-M1)
sbt:tapir> core / mimaPreviousArtifacts
[info] Set(com.softwaremill.sttp.tapir:tapir-core:0.18.0-M1)
```
In this case should we also ignore backwards compatibility?